### PR TITLE
Adding .NET Core 3.1 project samples

### PR DESCRIPTION
**Pull request by NinadKavimandan**

### CODE REVIEW

File: dotnet/aspnetcore31/containerWithTests/Application/aspnet-core-dotnet-core/Pages/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="content">
+        <div class="tweet-container">
+            <a href="http://twitter.com/intent/tweet/?text=I%20just%20created%20a%20new%20ASP.net%20website%20on%20Azure%20using%20Azure%20DevOps%20Project&hashtags=AzureDevOpsProject%2CVSTS%20%40Azure%20%40VSTS">
+                <img src="~/images/tweetThis.svg" />
+            </a>
+        </div>
+        <div class="content-body">
+            <div class="success-text">Success!</div>
+            <div class="description line-1"> Azure DevOps Project has been successfully setup</div>
+            <div class="description line-2"> Your ASP.NET CORE app is up and running on Azure</div>
```
Comment (bishal-pdMSFT) [inline dotnet/aspnetcore31/containerWithTests/Application/aspnet-core-dotnet-core/Pages/Index.cshtml:20]: `Core` instead of `CORE`

**Comment by NinadKavimandan** [comment]
Changes to look out for - 
1) TargetFramework version = netcoreapp3.1 for all projects,

2) For ASP.NET projects, Startup.cs to look like this - 
image

Migration steps followed from - https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.0&tabs=visual-studio

3) Updated 2.2 to 3.1 in all test cases and index.cshtml pages (wherever applicable).

4) URL to download dotnet hosting changes in ARM templates for VM based applications, check resource type "extensions".

File: dotnet/aspnetcore31/vm-webapp/ArmTemplates/windows-vm-template.json [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+          ],
+          "properties": {
+            "publisher": "Microsoft.Compute",
+            "type": "CustomScriptExtension",
+            "typeHandlerVersion": "1.9",
+            "autoUpgradeMinorVersion": true,
+            "protectedSettings": {
+              "commandToExecute": "[concat('powershell.exe setx /m APPINSIGHTS_INSTRUMENTATIONKEY ', reference(resourceId('microsoft.insights/components/', parameters('vmName')), '2015-05-01').InstrumentationKey, '; Import-Module servermanager; Add-WindowsFeature Web-Server; Invoke-WebRequest https://download.microsoft.com/download/0/1/D/01DC28EA-638C-4A22-A57B-4CEF97755C6C/WebDeploy_amd64_en-US.msi -outfile $env:temp\\WebDeploy_amd64_en-US.msi;  Start-Process $env:temp\\WebDeploy_amd64_en-US.msi -ArgumentList \"/quiet\" -Wait;  Invoke-WebRequest https://download.visualstudio.microsoft.com/download/pr/9b9f4a6e-aef8-41e0-90db-bae1b0cf4e34/4ab93354cdff8991d91a9f40d022d450/dotnet-hosting-3.1.6-win.exe -outfile $env:temp\\dotnet-hosting-3.1.6-win.exe;  Start-Process $env:temp\\dotnet-hosting-3.1.6-win.exe -ArgumentList \"/quiet\" -Wait;  net stop was /y;  net start w3svc')]"
```
Comment (bishal-pdMSFT) [inline dotnet/aspnetcore31/vm-webapp/ArmTemplates/windows-vm-template.json:254]: wow!! you figured out to change this command as well. Great :)

File: dotnet/aspnetcore31/containerWithTests/Application/aspnet-core-dotnet-core/Pages/Index.cshtml (hunk 1 above)
Comment (NinadKavimandan) [reply dotnet/aspnetcore31/containerWithTests/Application/aspnet-core-dotnet-core/Pages/Index.cshtml]: changed to Core

**Pull request merged**